### PR TITLE
Beeg Delta-station Floor-decal Retouches: SCIENCE

### PR DESCRIPTION
**Pull request by Cocacolagua**
## What Does This PR Do

Changes all floor-tile sprites in science to not look like shit.
Removes a lot of floor decals from science to not look like shit.
Changes some regular tables to reinforced tables to keep the same table style in the same room.
(Also moves around the stuff a lil' bit in toxins storage to make my life easier)

## Why It's Good For The Game
Current delta-station science looks horrible, as it spams horrible designs everywhere, you'll see what I mean.
In general, you want to have a simplistic design for the background, to not take too much focus away from the actual gameplay.
Delta-station suffers from this problem _everywhere_, I am starting my crusade against this ugly thing from here.

## Images of changes
TIME FOR EACH ROOM CHANGED WOO.

##
SCIENCE LOBBY BEFORE
PObzGbGRm5

SCIENCE LOBBY AFTER
Fqt1Yka0wM
##
RND BEFORE
1FkZsBaSze

RND AFTER
wcUsXwXYjm
##
MECH BAY BEFORE
36peRaa0HJ

MECH BAY AFTER
8jGhxqBlpS
##
ROBOTICS BEFORE
2yHqxUGGwZ

ROBOTICS AFTER
TrcYILhKmk
##
BREAK ROOM BEFORE
sVu5YZ1d7N

BREAK ROOM AFTER
chill room
##
SECURITY CHECKPOINT BEFORE (NO CHANGES)
yMlVnzaJhH

SECURITY CHECKPOINT AFTER (NO CHANGES)
sec checkpoint
##
EXPERIMENTOR BEFORE
R9aUO2rPFf

EXPERIMENTOR AFTER
pws1WTEl6V
##
XENOBIOLOGY BEFORE
AQnrv7nMCR
OGW332gXnT

XENOBIOLOGY AFTER
y7r7FjsyDL
##
CHECKPOINT BELOW SCIENCE LOBBY BEFORE
5A4cOO4cYF

CHECKPOINT BELOW SCIENCE LOBBY AFTER
checkpoint near lobby
##
RD OFFICE BEFORE
5DTlvv6UAv

RD OFFICE AFTER
nPUYzs3s4G
##
RND SERVERS BEFORE (NO CHANGES)
FnSuVmxlAC

RND SERVERS AFTER (NO CHANGES)
0zsyDPEEsQ
##
CHECKPOINT NEAR TOXINS BEFORE
9cCoEiopNl
CHECKPOINT NEAR TOXINS AFTER
checkpoint far lobby
##
TOXINS BEFORE
5IJsol3m2w

TOXINS AFTER
RRhHigSBwb
##
TOXINS LAUNCH BEFORE
CReIwVo7rR

TOXINS LAUNCH AFTER
toxins launch
##
TOXINS STORAGE BEFORE
0twmWBlFqJ

TOXINS STORAGE AFTER
toxins storage
##
WHATEVER THIS ROOM IS BEFORE
4u5GaOLTvi

WHATEVER THIS ROOM IS AFTER
what even is this

## Changelog
:cl:
tweak: Touches and recolors all the flooring and floor decals in delta-station's science department.
/:cl:

### CODE REVIEW

**Comment by SteelSlayer** [comment]
Honestly, I like some of the befores. Namely the cool purple patterns of the science lobby, hallways, toxins, etc. and also the mechbay. The mechbay does have too many warning stripes and whatnot, yes, but the after version feels a bit lifeless.

**Comment by DogsParadise** [comment]
I am not a massive fan of some befores they are super bussy and incosistent at times with patterns making no sense.

However.. they do have a certain pattern and logic they follow.

By just scrapping half of the room and not redoing it from the ground up, you are breaking the patterns. Making the rooms look incosistent.

For refrence, the placements of gaps here make 0 sense.

**Review by S34NW — CHANGES_REQUESTED** [review]
https://github.com/Cocacolagua/Paradise/pull/2

Please see my suggested changes in this PR on your branch, there are too many to put in comments here as images.
TLDR: Align all the edges, fix inconsistencies, touch up some areas that look cluttered

**Comment by Cocacolagua** [comment]
Changes made:
- The science floor style is now consistent, it will only change directions when reaching halfway across a room or crossing an object of interest (for the most part airlocks).
- Added some more dirt to less-used areas.
- RND holopad no longer breaks reality by being in the middle of two tiles.

**Comment by S34NW** [comment]
I know they do follow a pattern regarding doors and entryways, but they just look wrong and misaligned in these areas.
image
image

**Comment by Cocacolagua** [comment]
oh wait, for some reason the pictures didn’t change

**Review by S34NW — CHANGES_REQUESTED** [review]
If you are making extensive edits to delta science, now is the time to also fix up the piping there, please. Run pipes together, never under walls, and don't run pipes over vents/scrubbers if at all possible.

**Comment by Spacemanspark** [comment]
Can you add a roboticist's locker while you're at it

I'll give you a free fake pulse rifle if you do

**Comment by AffectedArc07** [comment]
Im torn on this.


On one hand, decal spam is dumb for maps. On the other hand, this makes delta delta.

**Comment by variableundefined** [comment]
Two maintainers objection. Main reason is that the uniqueness of the map is removed in favor of a cleaner aesthetics. We prefer the original aesthetics. 

Closing.
